Makefile

```diff
@@ -109,3 +109,18 @@ ifdef LAST_RELEASE_GIT_TAG
 else
 	$(error Cannot generate changelog without LAST_RELEASE_GIT_TAG)
 endif
+
+INTEGRATION_TESTS_SERVER_IMAGE    ?= hashicorppreview/consul:1.14-dev
+INTEGRATION_TESTS_DATAPLANE_IMAGE ?= $(PRODUCT_NAME)/release-default:$(VERSION)
+
+.PHONY: integration-tests
+integration-tests: docker/release-default
+ifdef INTEGRATION_TESTS_OUTPUT_DIR
+# make's built-in realpath function doesn't support non-existent directories
+# and intermittently has issues finding newly created ones (so preemptively
+# creating it with mkdir isn't an option) so we'll rely on the realpath bin.
+ifeq (, $(shell which realpath))
+ $(error "GNU Coreutils are required to run the integration-tests target with INTEGRATION_TESTS_OUTPUT_DIR.")
+endif
+endif
+	cd integration-tests && go test -v ./ -output-dir="$(shell realpath $(INTEGRATION_TESTS_OUTPUT_DIR))" -dataplane-image="$(INTEGRATION_TESTS_DATAPLANE_IMAGE)" -server-image="$(INTEGRATION_TESTS_SERVER_IMAGE)"
```

Comment:
Pardon my questions, still new at the company. 

Why does the server-image need to be defined here? Can the image already be defined in the Github config.yaml be used here as well? So there's no need to change the value in more than one place for future releases.

Reply:
Makes sense. I wonder if there's a way to inform anyone running this locally to change server image value to whatever the latest version is/find a way to keep this value up to date. I foresee this value becoming stale quickly.

Reply:
Definitely! I think once 1.14 ships we can just switch to the `latest` tag. Your local cache could still become stale, but it's a more sensible default, at least.